specification/resources/resource-manager/readme.go.md

```diff
@@ -208,6 +209,16 @@ namespace: locks
 output-folder: $(go-sdk-folder)/services/resources/mgmt/2015-01-01/locks
 ```
 
+### Tag: package-policy-2022-06 and go
+
+These settings apply only when `--tag=package-policy-2022-06 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-policy-2022-06' && $(go)
+namespace: policy
+output-folder: $(go-sdk-folder)/services/preview/resources/mgmt/2022-06-01-preview/policy
+```
+
```

Comment:
These configurations are for Go Track 1 SDK which will be deprecated very soon, therefore we do not need to update this file.
Could you please revert the changes here?